Uplift/Views/GymDetail/GymDetailTimeInfoView.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  GymDetailTimeInfoView.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 10/22/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class GymDetailTimeInfoView: UIView {
+
+    private let displayTextView = UITextView()
+
+    private var facilityDetail: FacilityDetail
+    private var selectedDayIndex = 0
+    private var displayedText = ""
+    private var timesText = NSMutableAttributedString()
+    private var paragraphStyle = NSMutableParagraphStyle()
+    private var paragraphStyleAttributes: [NSAttributedString.Key: Any] = [:]
+
+    // MARK: - Init
+    init(facilityDetail: FacilityDetail) {
+        self.facilityDetail = facilityDetail
+        super.init(frame: .zero)
+
+        timesText.mutableString.setString(displayedText)
+
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.lineSpacing = 5.0
+        paragraphStyle.alignment = .center
+        paragraphStyleAttributes = [
+            .font: UIFont._16MontserratRegular!,
```

Comment:
Have this be not a force unwrap? AFAIK Xcode will spit out a warning otherwise. I think you can do `UIFont._16MontserratRegular as Any` to disable that warning